tests/monotouch-test/example.pac

```diff
@@ -1,9 +1,9 @@
-// Exmaple PAC file that returns a proxy for the urls that have the
+﻿// Exmaple PAC file that returns a proxy for the urls that have the
```

Comment:
might as well fix `Exmaple` type if you're modifying this line

Reply:
Me an my typos..

(pun intended)